packages/hash/frontend/src/blocks/page/focusPageTitlePlugin.ts

```diff
@@ -0,0 +1,27 @@
+import { Schema } from "prosemirror-model";
+import { Plugin } from "prosemirror-state";
+
+export const focusPageTitlePlugin = new Plugin<any, Schema>({
+  props: {
+    handleKeyDown: (view, event) => {
+      const isArrowUp = event.key === "ArrowUp";
+
+      if (!isArrowUp) return false;
+
+      const { selection } = view.state;
+      const posAtIndex = selection.$head.posAtIndex(0, 1);
+
+      /** @todo `posAtIndex === 1` only works if first item is a text block, fix this */
+      if (posAtIndex === 1) {
+        const pageTitle =
+          document.querySelector<HTMLHeadingElement>("h1#hash-page-title");
+
+        pageTitle?.focus();
+
+        return true;
+      }
```

Comment:
So basically, if first block is a custom block like `Countdown `, not a text block like `Paragraph` or `Heading`, we cannot focus on first node on the ProseMirror document.

I have few ideas tol solve this. Let me know what do you think.

1. Implement focus on non-text blocks, to they can be selected as `NodeSelection`
2. Develop a work-around like "check if all of the previous nodes are non-text nodes, if yes, focus on header" (I don't like this idea)
3. Keep this as a known-issue for now.